frontend/packages/ui/src/icons/CardinalityZeroOrOneLeftIcon.tsx

```diff
@@ -7,26 +7,26 @@ export const CardinalityZeroOrOneLeftIcon: FC<Props> = (props) => {
     <svg
       role="img"
       aria-label="Cardinality Zero or One Left"
-      width={22}
+      width={23}
       height={16}
-      viewBox="0 0 22 16"
+      viewBox="0 0 23 16"
```

Comment:
Corrected the flipped direction of the cardinality icons (left and right).
![ss 2494](https://github.com/user-attachments/assets/627ce6c3-b00b-4738-b9fc-b25521978dcd)